src/coresyftools/coresyftools/gpt_tool.py

```diff
@@ -0,0 +1,101 @@
+'''Defines a CoReSyF Tool consisting of a SNAP single gpt operation.'''
+import os
```

Comment:
Well, this is debatable. By on hand, it is more convenient to just write `join` instead of `os.path.join`. By other hand, the ambiguity is very likely, because `join` (and other methods like these) can be a meaningful (and defined) method name in very diverse modules.

Reply:
Good point